client/app/theme/palette.js

```diff
@@ -35,20 +35,20 @@ const palette = {
   info: {
     contrastText: white,
     main: '#1070CA',
-    light: '#F1FBFC',
-    dark: '#007489',
+    // light: '#F1FBFC',
+    // dark: '#007489',
```

Comment:
Context: these colours were overriding the MUI defaults for the `<Alert>` component, and I don't see these light/dark colours used anywhere else.